File: Examples/test-suite/python/extern_c_runme.py
import extern_c

def check(flag):
    if not flag:
        raise RuntimeError("Test failed")

extern_c.RealFunction(2)
check(extern_c.cvar.int2 == 123)
check(extern_c.cvar.int3 == 456)
check(extern_c.cvar.int4 == 789)

File: Examples/test-suite/javascript/extern_c_runme.js
var extern_c = require("extern_c");

function check(flag) {
    if (!flag) {
        throw new Error("Test failed");
    }
}

extern_c.RealFunction(2);
check(extern_c.int2 == 123);
check(extern_c.int3 == 456);
check(extern_c.int4 == 789);
